python/cli/prelude_cli/views/partner.py

```diff
@@ -81,3 +81,14 @@ def partner_deploy(controller, partner, host_ids):
     with Spinner(description='Deploying probes to hosts'):
         data = controller.deploy(partner_code=Control[partner.upper()].value, host_ids=host_ids)
     print_json(data=data)
+
+
+@partner.command('webhook-auth')
+@click.argument('partner', type=click.Choice([c.name for c in Control if c != Control.INVALID], case_sensitive=False))
+@click.pass_obj
+@handle_api_error
+def feed_auth(controller, partner):
+    """ Get feed authentication information for a partner """
```

Comment:
can you add a better comment, more user-friendly?

Reply:
same with the command; i'd use something like ``webook-create`` or ``webhook-generate``. it's more descriptive of what you're doing in totality.. then i'd probably put a print line that points to our docs on how to finish the work/next steps.

Reply:
I think webhook-generate is a good option. I'll shift to that